apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -94,8 +90,8 @@ export class MarkdownDocumenter {
           documenter: new MarkdownDocumenterAccessor({
             getLinkForApiItem: (apiItem: ApiItem) => {
               return this._getLinkFilenameForApiItem(apiItem);
-            }
-          })
+            },
```

Comment:
I thought we didn't want trailing commas.

Reply:
I have updated **.prettierrc.js** based on our conversation